tasks/main.yml

```diff
@@ -17,6 +17,23 @@
     group: 'nginx'
     mode: 0775
 
+- name: Rotation for project nginx log file
+  logrotate:
+    name: '{{ application_name }}'
+    path: '/var/log/nginx/{{ application_name }}/*.log'
```

Comment:
I don't think it's a bad idea. Nginx logrotate also uses wildcard, so we don't have to update playbook if there is any other log file comes.